src/plot.jl

```diff
@@ -292,7 +292,7 @@ end
 
 
 # a function to update a value successively from the root to the tips
-function map_depthfirst(FUN, start, tree, eltype = nothing)
+function map_heightfirst(FUN, start, tree, eltype = nothing)
```

Comment:
Despite preferring heights for trees, I don't think anyone would fine `heightfirst` more intuitive than `depthfirst` for a search algorithm:
```suggestion
function map_depthfirst(FUN, start, tree, eltype = nothing)
```
It's not clear to me this function is used incidentally.